agnosterzak.zsh-theme

```diff
@@ -289,7 +289,7 @@ prompt_git() {
 
     prompt_segment $bgclr $fgclr
 
-    echo -n "%{$fg_bold[$fgclr]%}${ref/refs\/heads\//$PL_BRANCH_CHAR $upstream_prompt}${mode}$to_push$to_pull$clean$tagged$stashed$untracked$modified$deleted$added$ready_commit%{$fg_no_bold[$fgclr]%}"
+    echo -n " %{$fg_bold[$fgclr]%}${ref/refs\/heads\//$PL_BRANCH_CHAR $upstream_prompt}${mode}$to_push$to_pull$clean$tagged$stashed$untracked$modified$deleted$added$ready_commit%{$fg_no_bold[$fgclr]%} "
```

Comment:
One of you prefer the code styling instead of the convenience of the Users. I used back in the days this Zsh-git-prompt and i noticed that the Git prompt could stick out of the prompt on smaller screens like mine. Thanks for your app, i liked it.